### DISCUSSION TITLE
I don’t know if I should continue YouTubeEnhanced

### DISCUSSION THREAD

**Discussion by arichornlover** [arichorn/YouTubePlus#13] [General]
Well. I’m starting to have a feeling to not maintain this tweak because there is a New YouTube tweak that’s better then YouTubeEnhanced. there is this new tweak called YTLitePlus https://github.com/Balackburn/YTLitePlus and it’s better then YouTubeEnhanced, which is why I had this tweak archived. But now it’s unarchived in case if I continue or add in pull requests.

**Comment by chip707** [comment]
Thank you for this. I appreciate it.

**Comment by arichornlover** [comment]
> But now it’s unarchived in case if I continue or add in pull requests.

I will still do releases. I may be less active in this repository but this tweak will still get some cool changes. It’s not Discontinued since it wasn’t confirmed.
